### ISSUE TITLE
Gist: Error while contacting GitHub

### ISSUE THREAD

**Issue by faiwer** [condemil/Gist#153]
```
Traceback (most recent call last):
  File "request in /home/faiwer/.config/sublime-text-3/Installed Packages/Gist.sublime-package", line 135, in api_request
  File "request in /home/faiwer/.config/sublime-text-3/Installed Packages/Gist.sublime-package", line 65, in api_request_native
  File "./urllib/request.py", line 156, in urlopen
  File "./urllib/request.py", line 469, in open
  File "./urllib/request.py", line 492, in _open
  File "./urllib/request.py", line 447, in _call_chain
  File "./urllib/request.py", line 1310, in unknown_open
urllib.error.URLError: <urlopen error unknown url type: https>
error: Gist: Error while contacting GitHub
```

**Comment by jeremiahmarks** [comment]
Got to work today and installed this on a work computer and it just worked.  When I attempted to install it at home again, the same issue brought be back to this thread, so I decided to finally figure out what was up with this.  

This is actually an [issue with the embedded python](https://github.com/SublimeTextIssues/Core/issues/177) that sublime text comes with on linux. There seem to be a couple of workarounds, either [this solution](https://github.com/revolunet/sublimetext-markdown-preview/issues/27#issuecomment-11772098) which involves compiling a compatible python executable and replacing the one that comes bundled with sublime-text, or [this git](https://github.com/codexns/sublime-ssl-linux).  It seems like you would be able to do something with the files, however I have no idea how you would utilize them. (where to put them, how to execute them with relation to sublime-text, etc.)
